.circleci/verbatim-sources/linux-binary-build-defaults.yml

```diff
@@ -54,10 +57,11 @@ binary_linux_test: &binary_linux_test
   machine:
     image: ubuntu-1604:201903-01
   steps:
+  # See Note [Workspace for CircleCI scripts] in job-specs-setup.yml
   - attach_workspace:
       at: ~/workspace
-  - attach_workspace:
-      at: /home/circleci/project
```

Comment:
This is what's breaking https://github.com/pytorch/pytorch/blob/master/.circleci/scripts/binary_run_in_docker.sh#L33

Reply:
it should use https://github.com/pytorch/pytorch/blob/master/.circleci/scripts/binary_populate_env.sh#L87 , but just harcode it to the right thing for now.

We should really unify the ~/workspace and binary $workdir. I think it's not this way now cause ~/workspace == pytorch iiuc, but in binaries there's two repos so we need ~/workspace/pytorch and ~/workspace/builder